src/github.ts

```diff
@@ -112,6 +112,20 @@ export async function buildPiperFromSource (version: string): Promise<string> {
 
 export function getTag (version: string, forAPICall: boolean): string {
   version = version.toLowerCase()
-  if (version === '' || version === 'master' || version === 'latest') return 'latest'
+  if (version === '' || version === 'master' || version === 'latest') {
+    debug('Using latest tag')
+    return 'latest'
+  }
+  debug(`getTag returns: ${forAPICall ? 'tags' : 'tag'}/${version}`)
   return `${forAPICall ? 'tags' : 'tag'}/${version}`
 }
+
+export function getDownloadUrlByTag (version: string, forAPICall: boolean = false): string {
+  version = version.toLowerCase()
+  if (version === '' || version === 'master' || version === 'latest') {
+    debug('Using latest tag')
+    return `${GITHUB_COM_SERVER_URL}/SAP/jenkins-library/releases/latest`
+  }
+  debug(`getDownloadUrlByTag returns: ${GITHUB_COM_SERVER_URL}/SAP/jenkins-library/releases/${forAPICall ? 'tags' : 'tag'}/${version}`)
+  return `${GITHUB_COM_SERVER_URL}/SAP/jenkins-library/releases/${forAPICall ? 'tags' : 'tag'}/${version}`
+}
```

Comment:
simplify, maybe `url` needs to be defined outside the `if`.

```suggestion
    url = `${GITHUB_COM_SERVER_URL}/SAP/jenkins-library/releases/latest`
  } else {
    url = `${GITHUB_COM_SERVER_URL}/SAP/jenkins-library/releases/${forAPICall ? 'tags' : 'tag'}/${version}`
  }
  debug(`getDownloadUrlByTag returns: ${url}`)
  return url
}
```